azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -158,6 +158,11 @@ void setup(final ExecutableFlow flow) throws ExecutorManagerException {
       FileIOUtils.deleteDirectorySilently(tempDir);
       LOGGER.error("Error in preparing flow execution {}", flow.getExecutionId(), ex);
       throw new ExecutorManagerException(ex);
+    } catch (final Throwable t) {
+      FileIOUtils.deleteDirectorySilently(tempDir);
```

Comment:
This looks like a change in behavior. If a flow gets error in prepare stage, it wasn't cleaning up execution directory?

Reply:
Yes that is true.